pages/pipelines/managing_log_output.md.erb

```diff
@@ -143,3 +143,72 @@ For example, if you have environment variable `MY_SECRET="topsecret"`and you run
 <h3 class="Docs__note__heading">Setting environment variables</h3>
 <p>Note that if you <emphasis>set</emphasis> or <emphasis>interpolate</emphasis> a secret environment variable in your <code>pipeline.yml</code> it is not redacted, but doing that is <a href="/docs/pipelines/secrets#anti-pattern-storing-secrets-in-your-pipeline-dot-yml">not recommended</a>.</p>
 </div>
+
+## Private build log archive storage
+
+By default, build logs are stored in encrypted form in Buildkite's Amazon S3 buckets. If you want to have an increased level of security, you can store the archived build logs in your private AWS S3 bucket.
+
+<div class="Docs__note">
+<p>You need to be subscribed to the <a href="https://buildkite.com/pricing">Buildkite Enterprise</a> payment plan to use a private AWS S3 build log archive location.</p>
```

Comment:
Elsewhere we say "his feature is only available to Buildkite Enterprise users", we should be consistent about that.